libr/include/r_cons.h

```diff
@@ -822,6 +822,7 @@ R_API int r_cons_w32_print(const ut8 *ptr, int len, bool vmode);
 R_API int r_cons_win_printf(bool vmode, const char *fmt, ...);
 R_API int r_cons_win_eprintf(bool vmode, const char *fmt, ...);
 R_API int r_cons_win_vhprintf(DWORD hdl, bool vmode, const char *fmt, va_list ap);
+R_API void r_cons_win_set_cp(bool utf8);
```

Comment:
just wondering.. is there any reason why the win api is separated instead of being abstracted into a generic functionality for all platforms? .. something to think about and do in a separate pr